packages/web-pkg/src/composables/actions/useBlockedPopupRedirectMessage.ts

```diff
@@ -0,0 +1,14 @@
+import { useGettext } from 'vue3-gettext'
+
+export const useBlockedPopupRedirectMessage = () => {
+  const { $gettext } = useGettext()
+
+  return {
+    title: $gettext('Blocked pop-ups and redirects'),
```

Comment:
Please use a slightly different title to make clear, that blocking is not a feature but that the app detected, that pop ups are getting blocked: 
**"Pop-up and redirect block detected"**